frontend/src/employee-frontend/components/unit/tab-placement-proposals/PlacementProposalRow.tsx

```diff
@@ -57,20 +78,29 @@ type Props = {
     otherReason?: string
   ) => undefined | void
   loadUnitData: () => void
+  rejectedByCitizen: boolean
```

Comment:
The `placementPlan` is already in the props so having this as a separate field is unnecessary